### ISSUE TITLE
2.049 fails to build (Can't locate ./Types.pm at pdlperl.h.PL line 5.)

### ISSUE THREAD

**Issue by sebastic** [PDLPorters/pdl#315]
A parallel build issue seems to have been introduced between 2.043 and 2.049, the Debian package fails to build since it was updated to 2.049:

 https://buildd.debian.org/status/package.php?p=pdl&suite=experimental

From the [amd64 buildlog](https://buildd.debian.org/status/fetch.php?pkg=pdl&arch=amd64&ver=1%3A2.049-1%7Eexp1&stamp=1622641715&raw=0):
```
LD_RUN_PATH="" dh_auto_build
	make -j4
make[2]: Entering directory '/<<PKGBUILDDIR>>'
[...]
make[4]: Entering directory '/<<PKGBUILDDIR>>/Basic/Core'
"/usr/bin/perl" Types.pm.PL Types.pm 
"/usr/bin/perl" pdlperl.h.PL pdlperl.h 
Can't locate ./Types.pm at pdlperl.h.PL line 5.
make[4]: *** [Makefile:611: pdlperl.h] Error 2
make[4]: *** Waiting for unfinished jobs....
Types.pm.PL: using typedef long PDL_Indx
Extracting Types.pm
making PDL_B...
making PDL_S...
making PDL_US...
making PDL_L...
making PDL_IND...
making PDL_LL...
making PDL_F...
making PDL_D...
making PDL_CF...
making PDL_CD...
make[4]: Leaving directory '/<<PKGBUILDDIR>>/Basic/Core'
```
The package built fine on my system with `make -j3`.

**Comment by zmughal** [comment]
Duplicate of <https://github.com/PDLPorters/pdl/issues/314>.

**Comment by sebastic** [comment]
Confirmed fixed in 2.050:

 https://buildd.debian.org/status/package.php?p=pdl&suite=experimental

**Comment by mohawk2** [comment]
Thanks for feedback! Sorry for the inconvenience (and thanks to our heroic package maintainers for their vigilance :-)
